resources/checkstyle-exclude.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
You mean field level? (for variables and properties it should be possible as well)